controller/system_backup_controller.go

```diff
@@ -362,6 +365,14 @@ func (c *SystemBackupController) reconcile(name string, backupTargetClient engin
 		return
 	}
 
+	if backupTarget == nil && systemBackup.Status.State != longhorn.SystemBackupStateDeleting {
+		c.updateSystemBackupRecord(record,
+			systemBackupRecordTypeError, longhorn.SystemBackupStateError,
+			constant.EventReasonStart, "default backup target not found",
```

Comment:
If empty backupTarget remains and `State` is not `SystemBackupStateDeleting`, there will be flooding events. Right?